### DISCUSSION TITLE
Don't receive OTP by email

### DISCUSSION THREAD

**Discussion by lordkkjmix** [codemagic-ci-cd/codemagic-docs#1312] [Q&A] [answered]
Hello, 
I don't receive OTP on my email
Please help, i have already make too much config

**Comment by himesh-cm** [comment, marked as answer]
Hi @lordkkjmix, i am checking, i have edited the email from your post.

**Reply by himesh-cm** [reply]
Can you please try again? @lordkkjmix

**Reply by lordkkjmix** [reply]
it works thanks you so much
